go.mod

```diff
@@ -1,9 +1,11 @@
 module tinygo.org/x/drivers
 
+
 go 1.22.1
```

Comment:
I think this change slipped unintentionally here in previous MR - go version was supposedly be changed to `1.21` I guess (latest-2)? PTAL @deadprogram

Reply:
Probably not worth worrying about.